src/azure-cli/azure/cli/command_modules/acr/commands.py

```diff
@@ -112,6 +118,24 @@ def load_command_table(self, _):  # pylint: disable=too-many-statements
         operations_tmpl='azure.cli.command_modules.acr.check_health#{}'
     )
 
+    acr_scope_map_util = CliCommandType(
+        operations_tmpl='azure.cli.command_modules.acr.scope_map#{}',
+        table_transformer=scope_map_output_format,
+        client_factory=cf_acr_scope_maps
+    )
+
+    acr_token_util = CliCommandType(
+        operations_tmpl='azure.cli.command_modules.acr.token#{}',
+        table_transformer=token_output_format,
+        client_factory=cf_acr_tokens
+    )
+
+    acr_token_credential_generate_util = CliCommandType(
+        operations_tmpl='azure.cli.command_modules.acr.token#{}',
+        table_transformer=token_password_output_format,
```

Comment:
nit: rename to `token_credential_output_format`

Reply:
I think it was named that way since credentials consist of certificates and passwords. 
The method internally calls `_token_password_format_group` method since it deals with password formatting. I think it looks fine but I could change it if necessary.

https://github.com/AzureCR/azure-cli/blob/adupadhy/sm-token/src/azure-cli/azure/cli/command_modules/acr/_format.py#L69..L70